INSTALLATION.md

```diff
@@ -124,7 +136,7 @@ mv thyra-server* thyra-server && chmod +x ./thyra-server
 
 > **_NOTE:_** These commands should be executed directly from the directory where Thyra was downloaded.
 
-Congratulation, your version of Thyra is now installed on your system and can be run by executing `./thyra-server` in your terminal.
+Congratulation, your version of Thyra is now installed on your system and can be run by executing `thyra-server` in your terminal.
```

Comment:
In the manual installation the `thyra-server` has been moved to the current directory and so, contrary to the automatic, you have to make `./thyra-server`